deployment/sandbox-v2/terraform/azure/README.md

```diff
@@ -57,24 +59,79 @@ Generally, terraform communicates azure portal to create the resources through A
+    - Azure subscription ID
+    - admin password
+    - domain name label (Azure DNS name)
+    - NB: the Azure resource group name must *not* already exist.
+
+- In addition to the above changes to `variables.tf`, we need to make sure all resources are pointing to the same region. The location is specified in multiple files and in different ways. Take the following steps:
+    - In `variables.tf` change `location` to your azure region (e.g. from `"South India"` to `"UK South"`). Note the string syntax as Title Case instead of lowercase (e.g. `"southindia"`). 
+    - For each resource in `vm.tf`, `nic.tf`, and `vnet.tf`, there is a `location` field. This is often hardcoded as `"southindia"`. All instances should be set to `location = var.location` or your azure region so it picks it up automatically from the variables file. The string case type at this step does not seem to matter since Azure recognises Title Case as Azure regions.
+    - In `vm.tf` each virtual machine resource has a field `connection {host = ${var.domain_name_label}.southindia.cloudapp.azure.com}`.  The host needs to be changed to our location so that all resources are consistent, but crucially at this stage the string case *needs to be lowercase*. To resolve the discrepancy we create a new variable in `variables.tf`: `variable "locationlc" {default = "uksouth"}`. This then enables us to insert the variable into all the host strings as follows: `connection{host = ${var.domain_name_label}.${var.locationlc}.cloudapp.azure.com`.
+
+- MOSIP have hardcoded many '.sb' extensions, which may be appropriate for aws but is not appropriate for azure. In `mosip-infra/deployment/sandbox-v2/hosts.ini`, `mosip-infra/deployment/sandbox-v2/group_vars/mzcluster.yml`, `mosip-infra/deployment/sanbox-v2/playbooks/mzcluster.yml`, `mosip-infra/deployment/sanbox-v2/group_vars/dmzcluster.yml`, and `mosip-infra/deployment/sandbox-v2/playbooks/dmzcluster.yml` we need to remove all '.sb' extensions.
+
+#### Automated
+
+We have provided a script to automate the above changes. You can find it in `mosip-infra/deployment/sandbox-v2/turing_azure_edits.sh`.
+
+The script will prompt for user input specifying:   
+    - Azure subscription ID.  
+    - admin password.  
+    - domain name label (Azure DNS name).  
+    - Azure region (in Title Case).   
+
+If you have made a typo, you should be able to run the script again specifying the correct variable. 
 
- 3.	Once the **Azure CLI** is installed, enter **az login** command will provide a **URl and code** to authenticate with your account.
+To run, we assume you have the repository cloned. Since the file alters itself when it runs it's prudent to create a copy and run the copy instead:
+
+```
+$ cd ~
+$ cd mosip-infra/deployment/sandbox-v2
+$ cp ./turing_azure_edits.sh ./turing_azure_edits_copy.sh
+$ chmod u+x ./turing_azure_edits_copy.sh
+$ ./turing_azure_edits_copy.sh
+
+```     
+
```

Comment:
Perhaps after we've had enough of playing with our existing setup of MOSIP sandbox we can delete the VMs and start again to test the script